scripts/ros_release_python

```diff
@@ -80,7 +81,22 @@ def release_to_debian(name, version, debian_version, upload, ignore_upload_error
             '# ' + ' '.join(cmd) +
             (' [with %s]' % ' '.join(['%s=%s' % (k, v) for k, v in add_env.items()])
              if add_env else ''))
-        subprocess.check_call(cmd, env=dict(os.environ, **add_env))
+        # when doing a Python 2 release but no `python` executable is present
+        # use a temporary symlink to `python2` otherwise `stdeb` fails
+        symlink_python2 = python_version == 2 and not shutil.which('python') and shutil.which('python2')
```

Comment:
What happens when `python` exists and is a `python3` interpreter?

Reply:
That is not a supported case - not even by `stdeb`.

Reply:
Good to know. I guess I never would have run stdeb on Archlinux anyway.